DOCS/man/vo.rst

```diff
@@ -881,6 +881,15 @@ Available video output drivers are:
 
         Windows only.
 
+    ``dcomposition=<yes|no>``
+        Allows DirectComposition when using the ANGLE backend (default: yes).
+        Direct Composition is a Winows 8+ feature which improves rendering
```

Comment:
Though you'd probably be better at describing the issues than me, since it doesn't happen on my machine.